pom.xml

```diff
@@ -437,6 +437,18 @@
   </properties>
 
   <dependencies>
+
+    <dependency>
+      <groupId>com.nerdynick</groupId>
+      <artifactId>commons-configuration2-vault</artifactId>
+      <version>1.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.nerdynick</groupId>
+      <artifactId>commons-configuration-utils</artifactId>
```

Comment:
same question as before for this dependency.

Reply:
Same as above. It's a collection of config2 utils I've had assembled for a while now to just make things easier and quicker to do common patterns.